hash/oric1_cass.xml

```diff
@@ -262,10 +276,34 @@ license:CC0-1.0
 		</part>
 	</software>
 
+	<software name="arsenelp" supported="no">
+		<description>Arsène Lapin</description>
+		<year>1984</year>
+		<publisher>Infogrames</publisher>
+		<notes><![CDATA[
+Cassette with two versions, side 1 with the Atmos version, side 2 with Oric-1 version.
+]]></notes>
+		<info name="language" value="French" />
+		<info name="region" value="France" />
+		<part name="cass1" interface="oric1_cass">
+			<feature name="part_id" value="1 - Oric Atmos"/>
+			<dataarea name="cass" size="35157">
+				<rom name="arsenea.tap" size="35157" crc="b2a2a2c8" sha1="e4512ae9fbcff9fc9219086cf1f9405f2f7f51cc"/>
+			</dataarea>
+		</part>
+		<part name="cass2" interface="oric1_cass">
+			<feature name="part_id" value="2 - Oric 1 48K"/>
+			<dataarea name="cass" size="35198">
+				<rom name="arseneo.tap" size="35198" crc="e04acb4a" sha1="ade382af114c623b4e0d70f9f821a0073354fd11"/>
+			</dataarea>
+		</part>
+	</software>
+
 	<software name="asciichase">
 		<description>ASCII Chase</description>
 		<year>2017</year>
-		<publisher>Fabrizio Caruso</publisher>
+		<publisher>&lt;homebrew&gt;</publisher>
+		<info name="developer" value="Fabrizio Caruso" />
```

Comment:
I don’t understand the motivation for treating “homebrew” as a publisher.  If the developer distributes the software online themselves, isn’t it effectively self-published?  Can someone give a good reason for the fake “homebrew” publisher?

Reply:
If that's the rule, I made the changes.
I thought that if it was developed and self-published, the term "homebrew" would be appropriate.
(BTW, I think there are more software lists that use “homebrew” as a publisher)

Reply:
I know there are others.  I can understand the reason for using it for a system that wasn’t designed to allow self-hosted development, or was only designed to run licensed software.  For example the NES and Game Boy were intended to only run licensed software and had no provisions for users to develop their own software.  Homebrew software for these systems required the use of reverse-engineered development kits, Flash cartridges, etc. so it’s a completely different category to licensed cartridges, or even unlicensed and bootleg cartridges.

There’s also a distinction with the PlayStation, where homebrew software developed with the Net Yaroze system was far more restricted than licensed software.

But for an actual computer system, there’s effectively no difference.  For 8-bit home computers in particular, a lot of commercial software was developed by individuals, using the same tools available to everyone.  The distinction is completely artificial.